modules/monitoring-default-monitoring-components.adoc

```diff
@@ -0,0 +1,56 @@
+
+|Component|Description
+
+|Cluster Monitoring Operator
+|The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+
+|Prometheus Operator
+|The Prometheus Operator (PO) in the `openshift-monitoring` project creates, configures, and manages platform Prometheus instances and Alertmanager instances. It also automatically generates monitoring target configurations based on Kubernetes label queries.
+
+|Prometheus
+|Prometheus is the monitoring system on which the {product-title} monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+
+|Prometheus Adapter
+|The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and Pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal Pod autoscaling. The Prometheus Adapter is also used by the `oc adm top nodes` command.
+
+|Alertmanager
+|The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
+
+|`kube-state-metrics`
+|The `kube-state-metrics` exporter agent (KSM in the preceding diagram) converts Kubernetes objects to metrics that Prometheus can use.
+
+|`openshift-state-metrics`
+|The `openshift-state-metrics` exporter (OSM in the preceding diagram) expands upon `kube-state-metrics` by adding metrics for {product-title}-specific resources.
+
+|`node-exporter`
+|The `node-exporter` agent (NE in the preceding diagram) collects metrics about every node in a cluster. The `node-exporter` agent is deployed on every node.
+
+|Thanos Querier
+|The Thanos Querier aggregates and optionally deduplicates core {product-title} metrics and metrics for user-defined projects under a single, multi-tenant interface.
+
+|Grafana
+|The Grafana analytics platform provides dashboards for analyzing and visualizing the metrics. The Grafana instance that is provided with the monitoring stack, along with its dashboards, is read-only.
+
+|Telemeter Client
+|The Telemeter Client regularly polls core {product-title} Prometheus instances. The client federates a subsection of the data from those instances to Red Hat to facilitate Remote Health Monitoring for clusters.
+
+|===
+
+All of the components in the monitoring stack are monitored by the stack and are automatically updated when {product-title} is updated.
+
+[NOTE]
+====
+Each {product-title} component is responsible for its monitoring configuration. For problems with a component's monitoring, open a bug in Bugzilla against that component, not against the general monitoring component.
```

Comment:
in Bugzilla, we only have monitoring/telemeter component now, don't have each component, the note maybe is improper

Reply:
@juzhao I think this note refers to OCP components outside of the monitoring stack, for example `etcd`. The note comes from the previous '[Stack components and monitored targets](https://docs.openshift.com/container-platform/4.5/monitoring/cluster_monitoring/about-cluster-monitoring.html)' section. I will move the note in this PR to the 'Default monitoring targets' so that is is clear that this relates to components outside of the monitoring stack.

@christianh814 can you please clarify this?

Reply:
@adellape thank you for highlighting that. I have now resolved this on my local clone and will push the changes shortly.